internal/host/fioperfcheckcmd.go

```diff
@@ -0,0 +1,80 @@
+package host
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	FioPerfCheckCmdExitCode int64 = 126
```

Comment:
podman have 126 maybe something really big just to make sure that it doesn't conflict with any other error? 
https://github.com/containers/podman/issues/367

Reply:
Changed to 222